crates/ty_python_semantic/src/types.rs

```diff
@@ -4277,6 +4281,15 @@ impl<'db> Type<'db> {
         policy: MemberLookupPolicy,
     ) -> PlaceAndQualifiers<'db> {
         tracing::trace!("class_member: {}.{}", self.display(db), name);
+
+        // TODO: Test coverage for this.
+        if let Type::TypedDict(TypedDictType::Synthesized(synthesized_typed_dict)) = self
+            && let Some(member) =
+                TypedDictType::synthesized_member(db, self, synthesized_typed_dict.items(db), &name)
+        {
+            return Place::bound(member).into();
+        }
```

Comment:
I've copied this special case from https://github.com/astral-sh/ruff/pull/20732, but I'm not sure how to test it. What's a good way to access a member of a normalized type?

Reply:
I've expanded the TODO comment here.